modules/blogging/src/Volo.Blogging.Application/Volo/Blogging/Members/MemberAppService.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Threading.Tasks;
+using Volo.Abp.Domain.Repositories;
+using Volo.Blogging.Posts;
+using Volo.Blogging.Users;
+
+namespace Volo.Blogging.Members;
+
+public class MemberAppService : BloggingAppServiceBase, IMemberAppService
+{
+    private readonly IRepository<BlogUser, Guid> _userRepository;
+
+    public MemberAppService(IRepository<BlogUser, Guid> userRepository)
+    {
+        _userRepository = userRepository;
+    }
+    
+    public async Task<BlogUserDto> FindAsync(string username)
+    {
+        var user = await _userRepository.FindAsync(x => x.UserName == username);
+        
+        return ObjectMapper.Map<BlogUser, BlogUserDto>(user);
```

Comment:
You can make a null check here, and return null if it's not finding the record, otherwise, map to the `BlogUserDto` as you do now. (because, it throws `NullReferenceException` if the user could not be found and try to map the null value to `BlogUserDto`.